flocker/node/agents/test/test_cinder.py

```diff
@@ -61,3 +65,59 @@ def test_verify_false_ca_path(self):
             'verify_ca_path': '/a/path'
         }
         self.assertEqual(_openstack_verify_from_config(**config), False)
+
+
+class FakeCinderClient(object):
+    """"
+    Mock implementation of the cinder volume manager to use in the Cinder
```

Comment:
Not actionable, just FYI: for fakes we try to have verified fakes. That is, define interface that both fake and real client implement, and then run same tests against both implementation to ensure they act in same way. See e.g. `flocker.node._docker`, `flocker.node.test.test_docker` and `flocker.node.functional.test_docker`, where we have real and fake Docker clients tested by same interface compliance suite.

In this case I don't think it's worth the effort unless we decide OpenStack backend is higher priority and needs a lot more polish and in-depth testing.